test/unit/ngsiv2/contextRequests/TTModuleBLong.json

```diff
@@ -1,4 +1,8 @@
 {
+  "humidity":{
```

Comment:
Why this change was needed? Were test not passing without the change?

Reply:
Same reason explained in https://github.com/telefonicaid/iotagent-json/pull/339#discussion_r230902506